app/content/webviewPreload.js

```diff
@@ -2,10 +2,29 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-var ipc = require('electron').ipcRenderer
 var messages = require('../../js/constants/messages')
 var KeyCodes = require('../../js/constants/keyCodes')
 
+var ipc = {
+  events: {},
+  embedder: null,
+  processMessage: function (event) {
+    if (event.origin === 'file://') {
+      this.embedder = event.source
```

Comment:
No, you're right, it should only be set once. I had that in there, but I think I took it out when things weren't working right and apparently forgot to put it back.